src/components/config/index.ts

```diff
@@ -156,25 +196,20 @@ export const setupConfig = async (flags: any) => {
     )
   }
 
-  const parsedConfigs = await parseDefaultConfig(flags)
+  defaultConfigs = await parseDefaultConfig(flags)
 
-  let nonDefaultConfig: Partial<Config> | undefined
   if (flags.har) {
```

Comment:
Use switch-case instead?

Reply:
I don't think this is possible since switch evaluates expression instead of true-false values :sweat_smile: